src/components/datepicker/Datepicker.tsx

```diff
@@ -243,6 +245,10 @@ export const Datepicker = ({
     },
   };
 
+  if (showSkeleton) {
+    return <Skeleton animation height="38px" />;
```

Comment:
Can we use here scale975 ? :)

Reply:
Check, replaced it :). Also replaced it at some other components I copied it from. Think we didn't had the customsizing back then.